quantities/units/NonStandard/Mass/Pfund.cs

```diff
@@ -0,0 +1,12 @@
+using Quantities.Dimensions;
+
+namespace Quantities.Units.NonStandard.Mass;
+
+// https://de.wikipedia.org/wiki/Pfund
+public readonly struct Pfund : INoSystemUnit, IMass
```

Comment:
I never knew that the German "pound" differed from the English one. Nice trivia :)

Reply:
Yeah, working with these quantities is kind of fun. I sometimes also catch myself reading up on the history of some of them. They are a reflection of humanities journey in science :-)